### DISCUSSION TITLE
Future request: Chips left side icon

### DISCUSSION THREAD

**Discussion by CRC32EX** [primefaces/community#634] [PrimeReact] [answered]
I need [Chips](https://primereact.org/chips/) left side removable icon.
![image](https://github.com/primefaces/community/assets/25629617/f3226501-af8b-456d-9322-84c99e72a099)

For example, I have to move mouse every time to click icon.
If icon is left side, mouse doesn't need to move.

_GIF animation Example_
![](https://github.com/primefaces/community/assets/25629617/73b103cf-9d32-4166-9a2c-a3299baf4d50)

Chips `itemTemplate` atteribute does not support icon position now.

I also need [MultiSelect Chips](https://primereact.org/multiselect/#chips) left side icon.

**Comment by mertsincan** [comment, marked as answer]
Hi,

Could you please try;
```
.p-chips-token {
    gap: 0.5rem
}
.p-chips-token-icon {
   margin-left: 0;
   order: 1;
}
.p-chips-token-label {
   order: 2;
}
```
Best Regards,

**Reply by CRC32EX** [reply]
Works perfectly.
Thank you so much.
